src/test/scala/mesosphere/marathon/api/v2/AppsResourceTest.scala

```diff
@@ -341,6 +342,89 @@ class AppsResourceTest extends MarathonSpec with MarathonActorSupport with Match
     JsonTestHelper.assertThatJsonString(response.getEntity.asInstanceOf[String]).correspondsToJsonOf(expected)
   }
 
+  /*
+  test("Create a new app with IP/CT with virtual network foo w/ Docker") {
```

Comment:
The idea of the tests in this class is to exercise `AppsResource`'s methods with mocked dependencies, not with a proper `GroupManager`.

Do we already have a test in `GroupManager` that checks that it assign the right ports with an app like the one in this test? If the answer is yes, then we don't need this one. If it is not, then maybe we should add one there =).

I tried to make this test pass, but there are multiple problems with it, and I am not sure if we really need it.